src/ng/compile.js

```diff
@@ -3522,7 +3522,7 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
               }
               recordChanges(scopeName, newValue, oldValue);
               destination[scopeName] = newValue;
-            }, deepWatch);
```

Comment:
It probably makes sense to rename `deepWatch` now that isn't not used for deep-watching :smiley: